code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -2225,7 +2225,7 @@
 	to_chat(src, "<span class='xenodanger'>You feel your toxin glands refill, another young one ready for implantation. You can use Defile again.</span>")
 	update_action_button_icons()
 
-/mob/living/carbon/Xenomorph/proc/recurring_injection(mob/living/H, toxin1 = "xeno_toxin", toxin2 = null, larva = FALSE, count = 2)
+/mob/living/carbon/Xenomorph/proc/recurring_injection(mob/living/H, toxin1 = "xeno_toxin", toxin2 = null, larva = FALSE, virus1 = "necrovirus", count = 2)
```

Comment:
Is there need to use this proc? You can make your own.

Reply:
I suppose I can. Will make it more configurable. Will need a bit of time to figure it out though.